controllers/admin/admin_media_controller.py

```diff
@@ -26,6 +26,21 @@ def get(self):
         self.response.out.write(template.render(path, self.template_values))
 
 
+class AdminMediaDeleteReference(LoggedInHandler):
+    def post(self, media_key_name):
+        self._require_admin()
+
+        media = Media.get_by_id(media_key_name)
+
+        media.references.remove(media.create_reference(
+            self.request.get("reference_type"),
+            self.request.get("reference_key_name")))
+
+        media.put() ## Todo, update manipulators to support removing an item that ends a list
```

Comment:
I tried to do this and hit a roadblock that's going to require changing `BaseManipulator`.

To get a manipulator to *remove* items from a list (like `Media.references`), you have to call `MediaManipulator.createOrUpdate(...,auto_union=False)`. CreateOrUpdate ends up calling in to `MediaManipulator.updateMerge()`, where the default behavior is to merge lists. auto_union false will replace the old list with the new list.

However, then the `list_attr` comparer section only replaces a list if the new list has `len > 0`: https://github.com/the-blue-alliance/the-blue-alliance/blob/master/helpers/media_manipulator.py#L46

To get around this, we can:

1. Modify `MediaManipulator` to replace populated list_attr's with empty list list_attrs all the time
2. Modify `MediaManipulator` to replace populated list_attr's with empty list list_attrs when `auto_union == False`
3. Modify `ManipulatorBase` to thread a `consider_empty_lists` flag through all the method calls

I am not sure where else in code we may be relying on Manipulators to not clobber populated lists, so I am apprehensive to do anything except the 3rd option, but that felt clunky as I started to do it.

Any better ideas?